src/components/shared/Validations.tsx

```diff
@@ -108,13 +107,8 @@ function filterOutOasFormatValidations(format: string, values: Dictionary<unknow
 
 export const Validations: React.FunctionComponent<IValidations> = ({ validations, hideExamples }) => {
   const numberValidations = pick(validations, numberValidationNames);
-  const booleanValidations = omit(
```

Comment:
you prob want to revert it

Reply:
that excludes it both from `booleanValidations` and `keyValueValidations` but I want them to still show in keyVal ones. But I can fix that. I'm just wondering, Is there any usecase where it makes more sense to use `NameValidations`  rather than `KeyValueValidations`?

Because, if the value is `true` or `false` I think it makes more sense to display it using `KeyValueValidations`, so it says `default: true` rather than `Default`. Same for `example` or even for `exclusiveMinimum` (which wont be shown here because it falls in `numberValidations`)

Reply:
> Is there any usecase where it makes more sense to use NameValidations rather than KeyValueValidations?

That's a good question... I'm not really sure whether we actually have any booleanish validations left now that we filter out default, readOnly, writeOnly, and everything related to numbers is handled separately. 

>  I think it makes more sense to display it using KeyValueValidations, so it says default: true rather than Default

I agree

Reply:
We used NameValidations before because some of they sounded better this way (the property name was enough) it ie. deprecated, readOnly, writeOnly etc. Default can be any type, but agree that as boolean it might look weird. If it makes more sense to change it given those validations are represented differently now then go ahead, I think it's fine.

Reply:
Gotcha. Yea I think most of them are handled differently now (all those mentioned included)